test/JDBC/upgrade/14_5/schedule

```diff
@@ -36,6 +36,7 @@ TestUDD
 TestUniqueIdentifier
 TestVarChar
 TestXML
+forxml-before-14-7
```

Comment:
Let's rename this to `forxml-before-14-7-or-15-2` to have the same test names between 2_X_DEV and 3_X_DEV.